src/main/java/cofh/thermaldynamics/duct/light/LightGrid.java

```diff
@@ -114,7 +115,7 @@ public void setLight(boolean lit) {
 				for (ChunkCoord chunk : chunks) {
 
 					PlayerChunkMapEntry inst = manger.getEntry(chunk.chunkX, chunk.chunkZ);
-					if (inst != null && inst.players.contains(player)) {
+					if (inst != null && inst.containsPlayer((EntityPlayerMP) player)) {
```

Comment:
context: PlayerChunkMapEntry#players is private by default. We can use the method `containsPlayer()` instead so we dont need an AT.